### ISSUE TITLE
i installed plugin and it's not showing up on octopint plugin list

### ISSUE THREAD

**Issue by bogaskilop** [kontakt/Octoprint-Filament-Reloaded#36]
i installed plugin and it's not showing up on octopint plugin list

how can i configure this plugin ?r

**Comment by megabri** [comment]
Are you perform a restart of the Octoprint Server ?
Also in my case the same, but the plugin need a restart to be showed in the plugin list.

**Comment by bogaskilop** [comment]
yes i always do restart after new plugin installation most of them are forced to restart octoprint server also

**Comment by bolsoncerrado** [comment]
Same issue. A whole reboot fixed it.
